samples/snippets/src/test/java/com/example/spanner/SampleTestBase.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+/**
+ * Base class for sample integration tests.
+ */
+public class SampleTestBase {
```

Comment:
Had to use inheritance here =(.

@olavloite let me know if that is ok or if you know of any better way to enforce `beforeClass` / `afterClass` in the tests.

Reply:
As far as I know, this is the best way to do that in Junit (4).

Reply:
Thanks for confirming.